src/server/webpack/modules.js

```diff
@@ -1,15 +1,16 @@
+const log = require('loglevel')
 const path = require('path')
 const fs = require('fs')
-const { COMPONENT_ABSOLUTE_PATH_DIR } = require('../../constants')
 
-const get = async () => {
+const get = async (component) => {
   const paths = []
-  let previousPath = COMPONENT_ABSOLUTE_PATH_DIR
+  let previousPath = component.path.absolute
   let files = await fs.readdirAsync(previousPath)
```

Comment:
`yarn start` doesn't work anymore.
Because, here `modules.js` is waiting for the component **folder** and not the component **file** path.